Python/SVY21.py

```diff
@@ -143,17 +148,50 @@ def computeLatLon(self, N, E):
         # Compute Latitude
         latFactor = tPrime / (self.k * rhoPrime)
         latTerm1 = latFactor * ((Eprime * x) / 2)
-        latTerm2 = latFactor * ((Eprime * x3) / 24) * ((-4 * psiPrime2) + (9 * psiPrime) * (1 - tPrime2) + (12 * tPrime2))
-        latTerm3 = latFactor * ((Eprime * x5) / 720) * ((8 * psiPrime4) * (11 - 24 * tPrime2) - (12 * psiPrime3) * (21 - 71 * tPrime2) + (15 * psiPrime2) * (15 - 98 * tPrime2 + 15 * tPrime4) + (180 * psiPrime) * (5 * tPrime2 - 3 * tPrime4) + 360 * tPrime4)
+        latTerm2 = latFactor * ((Eprime * x3) / 24) * (
+                (-4 * psiPrime2) + (9 * psiPrime) * (1 - tPrime2) + (12 * tPrime2))
+        latTerm3 = latFactor * ((Eprime * x5) / 720) * (
+                (8 * psiPrime4) * (11 - 24 * tPrime2) - (12 * psiPrime3) * (21 - 71 * tPrime2) + (
+                15 * psiPrime2) * (15 - 98 * tPrime2 + 15 * tPrime4) + (180 * psiPrime) * (
+                        5 * tPrime2 - 3 * tPrime4) + 360 * tPrime4)
         latTerm4 = latFactor * ((Eprime * x7) / 40320) * (1385 - 3633 * tPrime2 + 4095 * tPrime4 + 1575 * tPrime6)
         lat = latPrime - latTerm1 + latTerm2 - latTerm3 + latTerm4
 
         # Compute Longitude
         secLatPrime = 1. / math.cos(lat)
         lonTerm1 = x * secLatPrime
         lonTerm2 = ((x3 * secLatPrime) / 6) * (psiPrime + 2 * tPrime2)
-        lonTerm3 = ((x5 * secLatPrime) / 120) * ((-4 * psiPrime3) * (1 - 6 * tPrime2) + psiPrime2 * (9 - 68 * tPrime2) + 72 * psiPrime * tPrime2 + 24 * tPrime4)
+        lonTerm3 = ((x5 * secLatPrime) / 120) * ((-4 * psiPrime3) * (1 - 6 * tPrime2) + psiPrime2 * (
+                9 - 68 * tPrime2) + 72 * psiPrime * tPrime2 + 24 * tPrime4)
         lonTerm4 = ((x7 * secLatPrime) / 5040) * (61 + 662 * tPrime2 + 1320 * tPrime4 + 720 * tPrime6)
         lon = (self.oLon * math.pi / 180) + lonTerm1 - lonTerm2 + lonTerm3 - lonTerm4
 
+        #X IS LON AND Y IS LAT
         return (lat / (math.pi / 180), lon / (math.pi / 180))
+
+
+def main():
```

Comment:
Thanks for shifting this into its own function. Could you give this function a more descriptive name (like `csvConverter`), as well as some comments on what it does?
Then, to use this file as a standalone script, you can add this to the bottom of this file (it would be great if you could add some documentation too):
```
# Converts blah blah... to blah blah...
# Usage: python SVY21.py file1.csv file2.csv
if __name__ == '__main__':
    csvConverter()
```

Reply:
Hey thanks for the advice! But i'm not too sure what kind of documentation I can write.. Hope what I have is ok.